pkg/broker/handler/processors/deliver/processor_test.go

```diff
@@ -235,16 +246,33 @@ func (h *targetWithFailureHandler) ServeHTTP(w http.ResponseWriter, req *http.Re
 		w.Header().Del("ce-specversion")
 		// Content-Type does not start with `application/cloudevents`
 		w.Header().Set("Content-Type", "text/html")
+	} else if h.structuredContentMode {
+		w.Header().Set("Content-Type", "application/cloudevents+json; charset=utf-8")
 	}
 	w.WriteHeader(h.respCode)
 	w.Write([]byte(h.respBody))
 }
 
+type testingReplyHandler struct {
```

Comment:
Reply seems too generic for understanding, maybe change to sth like `brokerReplyHandler` or `brokerIngressReplyHandler`.

Reply:
Renamed it and added a description. I don't want to use `broker` in the name because this already generic for all CellTenants.